src/model/document.js

```diff
@@ -306,6 +293,44 @@ export default class Document {
 		return json;
 	}
 
+	/**
+	 * Check if there were any changes done on document, and if so, call post-fixers,
+	 * fire `change` event for features and conversion and then reset the differ.
+	 * Fire `change:data` event when at least one operation or buffered marker changes the data.
+	 *
+	 * @protected
+	 * @fires change
+	 * @fires change:data
+	 * @param {module:engine/model/writer~Writer writer} writer The writer on which post-fixers will be called.
+	 */
+	_runPostFixersAndResetDiffer( writer ) {
```

Comment:
```suggestion
	_handleChange( writer ) {
```

Reply:
I do not like the current name. The most important thing this method does is firing `document#change`. However, `_runPostFixersAndFireChangeAndResetDiffer` sounds ridiculous, so I am for shorter `_handleChangeBlock `